fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalDistributionOperator.java

```diff
@@ -46,6 +48,20 @@ public <R, C> R accept(OptExpressionVisitor<R, C> visitor, OptExpression optExpr
 
     @Override
     public boolean couldApplyStringDict(Set<Integer> childDictColumns) {
+        final DistributionSpec distributionSpec = getDistributionSpec();
+        if (distributionSpec instanceof HashDistributionSpec) {
+            HashDistributionSpec hashDistributionSpec = (HashDistributionSpec) distributionSpec;
+            final HashDistributionDesc desc = hashDistributionSpec.getHashDistributionDesc();
+            if ((desc.isShuffle() || desc.isShuffleEnforce() || desc.isBucketJoin()) && !desc.isAggShuffle()) {
```

Comment:
Why !desc.isAggShuffle() ?

Reply:
desc.isShuffle() equels SHUFFLE_JOIN and SHUFFLE_AGG.
It's OK to process SHUFFLE_AGG